src/remux/mp4-remuxer.js

```diff
@@ -318,7 +325,7 @@ class MP4Remuxer {
       }
     }
 
-    if (chromeVersion && chromeVersion < 75) {
+    if (this.requiresPositiveDts) {
```

Comment:
Unnecessary `this` (`requiresPositiveDts` is not instance variable) ?

Reply:
Thanks for the catch @Korilakkuma!

```suggestion
    if (requiresPositiveDts) {
```